duo-client/src/main/java/com/duosecurity/client/canonicalization/V2.java

```diff
@@ -0,0 +1,20 @@
+package com.duosecurity.client.canonicalization;
+
+public class V2 extends Canonicalizer {
+    private String date;
+
+    public V2(String inDate) {
```

Comment:
For example, v2 compared to v1 require date, so v2 need String inDate as constructor.